packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -314,6 +338,8 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "nextPageTooltip": r"Nächste Seite",
     "previousPageTooltip": r"Vorherige Seite",
     "showMenuTooltip": r"Menü anzeigen",
+    "aboutListTileTitle": r"Über $applicationName",
+    "licensesPagePoweredByTitle": r"Angetrieben durch Flutter",
```

Comment:
That's a tough one. I don't think German has a good translation for this. In fact, most hip German apps would probably just use the english "Powered by xyz" here. If you are against using Denglish (mix of Deutsch and English), you could say something like "Mit Flutter gebaut" (which translates back to "built with Flutter").

Reply:
Mit Flutter gebaut sounds good to me!